src/pdf.worker.js

```diff
@@ -12,13 +12,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-/* eslint-disable no-unused-vars */
 
-"use strict";
```

Comment:
Why exactly is this removed (here and in the `pdf.js` file)?

Reply:
Because using native `import` statements will automatically put the code into strict mode, according to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import, and looking at the built files (the `mozcentral` build target) there's also still `"use strict";` lines everywhere as expected.